diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 5390af3..46514a6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -125,7 +125,7 @@ endmacro()
 
 if (SWIG_FOUND)
   if(BUILDSWIGPYTHON)
-    find_package (PythonLibs)
+    find_package (PythonLibs 3.3)
     string (REPLACE "." ";" PYTHON_VERSION_LIST ${PYTHONLIBS_VERSION_STRING})
     list (GET PYTHON_VERSION_LIST 0 PYTHON_VERSION_MAJOR)
     list (GET PYTHON_VERSION_LIST 1 PYTHON_VERSION_MINOR)
