diff --git a/cpp/build/support.mk b/cpp/build/support.mk
index b76fd1f..71b3d7b 100644
--- a/cpp/build/support.mk
+++ b/cpp/build/support.mk
@@ -16,7 +16,7 @@ DOXYGEN := $(shell which doxygen)
 #dot is required for doxygen (part of Graphviz)
 DOT := $(shell which dot)
 #the machine type we are building on (i686 or x86_64)
-MACHINE := $(shell uname -m)
+MACHINE := i686
 #the location of xmllink for checking our config files
 XMLLINT := $(shell which xmllint)
 #temp directory to build our tarfile for make dist target
@@ -101,11 +101,7 @@ instlibdir ?= $(PREFIX)$(instlibdir.default)
 endif
 
 #pkg-config doesn't exist, lets try to guess best place to put the pc file
-ifeq ($(PKGCONFIG),)
-pkgconfigdir ?= $(shell if [ -d "/usr/lib64/pkgconfig" ]; then echo "/usr/lib64/pkgconfig"; else echo "/usr/lib/pkgconfig"; fi)
-else
-pkgconfigdir ?= $(shell test -d "$(instlibdir)/pkgconfig" && echo "$(instlibdir)/pkgconfig" || pkg-config --variable pc_path pkg-config | awk -F: '{ print $$1 }')
-endif
+pkgconfigdir ?= /usr/lib/pkgconfig
 
 ifeq ($(BITBAKE_ENV),1)
 sysconfdir := $(PREFIX)/etc/openzwave/
