--- a/cpp/examples/MinOZW/Makefile	2015-12-21 21:55:19.602443460 +0100
+++ b/cpp/examples/MinOZW/Makefile	2015-12-21 21:54:19.874445019 +0100
@@ -49,7 +49,7 @@
 
 $(OBJDIR)/MinOZW:	$(patsubst %.cpp,$(OBJDIR)/%.o,$(minozwsrc))
 	@echo "Linking $(OBJDIR)/MinOZW"
-	$(LD) $(LDFLAGS) $(TARCH) -o $@ $< $(LIBS) -pthread
+	$(CXX) $(LDFLAGS) $(TARCH) -o $@ $< $(LIBS) -pthread
 
 $(top_builddir)/MinOZW: $(top_srcdir)/cpp/examples/MinOZW/MinOZW.in $(OBJDIR)/MinOZW
 	@echo "Creating Temporary Shell Launch Script"
